### DISCUSSION TITLE
6.0 wont load but 5.2 will.

### DISCUSSION THREAD

**Discussion by HubFly35** [Khopa/dcs_liberation#2645] [Q&A] [answered]
I can't seem to start Liberation 6.0. When I run the main exe. it will throw the splash screen (the Ghost of Kyiv) up and then "Try" to load the map/interface but everything stays blank/white. When I click on it, my windows says the program is unresponsive, would you like to wait or close. My computer stays fine and I can do other things with it but the liberation exe just keeps waiting. 

However, when I try to load the 5.2 version...not problems and it loads super quick. 

I have never had this issue before. I have deleted all liberation versions and reinstalled but same issues. 

Any suggestions would be hugely appreciated. Thanks!

![1](https://user-images.githubusercontent.com/120773303/208350301-34b512cb-8deb-4637-8460-3c29dcff790d.JPG)

**Comment by Starfire13** [comment]
Make sure you're deleting the stuff in C:\Users\ <username> \AppData\Local\dcsliberation\ as well when reinstalling. Apart from that, check that your antivirus isn't automatically quarantining the executable (some antivirus programs flag Liberation as a false positive).

**Comment by HubFly35** [comment]
I tried both with no result sadly.

**Comment by Starfire13** [comment]
Check under C:\ProgramData\Microsoft\Windows\WER\ReportArchive\ for app crash info.

**Comment by HubFly35** [comment]
I just looked and sure enough there is crash data there. 

It is classifying it as "AppHangB1"??? Ultimately it says: "ReportDescription=A problem caused this program to stop interacting with Windows." It doesn't list or say where it failed or broke down. 

Would it help to drop the file here? Is there anything I should scrub (privacy and security wise before it do? 

Thank you again!

**Reply by Starfire13** [reply, marked as answer]
If it doesn't state the reason for the crash, it might not be of much use. Ultimately, this might not be something we can help much with because it's specific to your system. It is just on your system that Liberation is crashing. So, you might just have to troubleshoot it like you would any other crash. What I will say is that on the very rare occasions others have encountered crashes like this, it has quite often turned out to be some third-party antivirus software or other that is the cause.

**Comment by jcllore** [comment]
The exact same thing happens to me, but it happens to me when I plan the flights, which does not happen to me with version 5.2
